content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -82,44 +77,49 @@ rules:
-It holds a list of subjects (users, groups, or service accounts), and a reference to the role being granted.
-Permissions can be granted within a namespace with a `RoleBinding`, or cluster-wide with a `ClusterRoleBinding`.
+It holds a list of *subjects* (users, groups, or service accounts), and a reference to the
+role being granted.
+A RoleBinding grants permissions within a specific Namespace whereas a ClusterRoleBinding
+grants that access cluster-wide.
 
-A `RoleBinding` may reference a `Role` in the same namespace.
-The following `RoleBinding` grants the "pod-reader" role to the user "jane" within the "default" namespace.
-This allows "jane" to read pods in the "default" namespace.
-
-`roleRef` is how you will actually create the binding.  The `kind` will be either `Role` or `ClusterRole`, and the `name` will reference the name of the specific `Role` or `ClusterRole` you want. In the example below, this RoleBinding is using `roleRef` to bind the user "jane" to the `Role` created above named `pod-reader`.
+A RoleBinding may reference any Role in the same Namespace.
+Here is an example of a RoleBinding that grants the "pod-reader" Role to the user "jane"
+within the "default" Namespace.
+This grant allows "jane" to read Pods in the "default" namespace.
 
 ```yaml
 apiVersion: rbac.authorization.k8s.io/v1
 # This role binding allows "jane" to read pods in the "default" namespace.
+# You need to already have a Role named "pod-reader" in that namespace.
 kind: RoleBinding
 metadata:
   name: read-pods
   namespace: default
 subjects:
+# You can specify more than one "subject"
 - kind: User
-  name: jane # Name is case sensitive
+  name: jane # "name" is case sensitive
   apiGroup: rbac.authorization.k8s.io
 roleRef:
+  # "roleRef" specifies the binding to a Role / ClusterRole
   kind: Role #this must be Role or ClusterRole
   name: pod-reader # this must match the name of the Role or ClusterRole you wish to bind to
   apiGroup: rbac.authorization.k8s.io
 ```
 
-A `RoleBinding` may also reference a `ClusterRole` to grant the permissions to namespaced
-resources defined in the `ClusterRole` within the `RoleBinding`'s namespace.
-This allows administrators to define a set of common roles for the entire cluster,
-then reuse them within multiple namespaces.
+A RoleBinding can reference a ClusterRole to grant the permissions defined in that
+ClusterRole to resources inside the RoleBinding's namespace. This kind of reference
+lets you define a set of common roles across your cluster, then reuse them within
+multiple namespaces.
 
-For instance, even though the following `RoleBinding` refers to a `ClusterRole`,
-"dave" (the subject, case sensitive) will only be able to read secrets in the "development"
-namespace (the namespace of the `RoleBinding`).
+For instance, even though the following RoleBinding refers to a ClusterRole,
+"dave" (the subject, case sensitive) will only be able to read Secrets in the "development"
```

Comment:
prefer lowercase plural "secrets" resource, since that is what appears in the role definition